app/controllers/submitted_order_selection.js

```diff
@@ -15,6 +15,32 @@ export default Ember.Controller.extend(asyncTasksMixin, {
     return this.get("cart.isEmpty");
   },
 
+  submittableOrders: Ember.computed("model.@each.state", function() {
+    let submittableOrders = this.get("model").filter(
+      this.isPackageSubmittableOrder
+    );
+    if (submittableOrders.length) {
+      return submittableOrders;
+    } else {
+      this.redirectToRequestPurpose();
+    }
+  }),
+
+  redirectToRequestPurpose() {
+    this.transitionToRoute("request_purpose", {
+      queryParams: {
+        onlineOrder: true,
```

Comment:
one of these days we'll need to clean this a bit, having 2 opposite flags `onlineOrder/bookAppointment` is bound to eventually cause us problems (maybe we can move to a `orderType: "appointment"` format).

This is unrelated to this PR, just thought I'd mention it 😄